wirepas_mqtt_library/wirepas_network_interface.py

```diff
@@ -271,6 +344,18 @@ def wrapper(*args, **kwargs):
         wrapper.__doc__ = fn.__doc__
         return wrapper
 
+    def release(self):
```

Comment:
A method like this is canonically called `close()`. Many built-in Python types, e.g. files and sockets call it that.